cfme/services/dashboard/ssui.py

```diff
@@ -47,21 +47,24 @@ def is_displayed(self):
         return self.in_myservices and self.title.text == "My Services"
 
 
-@Dashboard.total_service.external_implementation_for(ViaSSUI)
-def total_service(self):
-    """Returns the count of total services(Integer) displayed on dashboard"""
+@Dashboard.total_services.external_implementation_for(ViaSSUI)
+def total_services(self):
```

Comment:
Also I would say that such check belongs to test, consider if it wouldn't be better if this really just returned number of displayed services. Such behavior would also be consistent with e.g. ``total_requests``.

Or consider changing the function name to reflect that it's checking the number of services, not returning their number.